sampling/hierarchical/README.md

```diff
@@ -0,0 +1,42 @@
+# opentelemetry-sampling-hierarchical
+
+The `opentelemetry-sampling-hierarchical` gem allows for specifying sampling frequencies based on the hierarchical invocation of other samplers.
+
+## What is OpenTelemetry?
+
+[OpenTelemetry][opentelemetry-home] is an open source observability framework, providing a general-purpose API, SDK, and related tools required for the instrumentation of cloud-native software, frameworks, and libraries.
+
+OpenTelemetry provides a single set of APIs, libraries, agents, and collector services to capture distributed traces and metrics from your application. You can analyze them using Prometheus, Jaeger, and other observability tools.
+
+## How does this gem fit in?
+
+This gem can be used with any OpenTelemetry SDK implementation. This can be the official `opentelemetry-sdk` gem or any other concrete implementation.
+
+## How do I get started?
+
+Install the gem using:
+
+```
+gem install opentelemetry-sampling-hierarchical
+```
+
+Or, if you use [bundler][bundler-home], include `opentelemetry-sampling-hierarchical` in your `Gemfile`.
+
```

Comment:
I think it would be helpful to add example usage in the README and/or an `example` directory with a script that shows how to configure it.

Reply:
Unfortunately, there is no way yet to use it. This will require an update to https://github.com/open-telemetry/opentelemetry-ruby/blob/d365dfeb3982f09cd62a17c6e67eb3792f56e460/sdk/lib/opentelemetry/sdk/trace/tracer_provider.rb#L167. I will open a PR to support the [X-Ray sampler](https://github.com/open-telemetry/opentelemetry-ruby-contrib) there as well and will adjust the README afterwards accordingly if that's fine with you